### PR TITLE
Remove highlighting when focusing on a carousel button

### PR DESCRIPTION
**Pull request by pe4cey**

### CODE REVIEW

File: src/browser/components/buttons/index.jsx [hunk 1]
```diff
@@ -290,13 +290,22 @@ const BaseCarouselButton = styled.button`
   width: 40px;
   min-width: 40px;
   -webkit-font-smoothing: antialiased;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none;
```
Comment (oskarhane) [inline src/browser/components/buttons/index.jsx]: `styled-components` generates vendor prefix, you don't have to.
See generated CSS below: 
oskar4j 2017-04-11 at 13 15 06

Comment (pe4cey) [inline src/browser/components/buttons/index.jsx]: @oskarhane that's good to know. Updated.

File: src/browser/components/buttons/index.jsx [hunk 2]
```diff
@@ -289,14 +289,17 @@ const BaseCarouselButton = styled.button`
   top: 130px;
   width: 40px;
   min-width: 40px;
-  -webkit-font-smoothing: antialiased;
```
Comment (oskarhane) [inline src/browser/components/buttons/index.jsx]: I think we want to keep this one though.

Comment (pe4cey) [inline src/browser/components/buttons/index.jsx]: Updated

**Pull request merged**